src/nft/components/bag/BagFooter.tsx

```diff
@@ -226,7 +225,7 @@ export const BagFooter = ({
               <ThemedText.SubHeaderSmall>
                 <Trans>Pay with</Trans>
               </ThemedText.SubHeaderSmall>
-              <CurrencyInput onClick={toggleTokenSelector}>
+              <CurrencyInput onClick={() => setTokenSelectorOpen(true)}>
                 <CurrencyLogo currency={activeCurrency} size="24px" />
                 <ThemedText.HeadlineSmall fontWeight={500} lineHeight="24px">
                   {activeCurrency?.symbol}
```

Comment:
Large token values wrap outside the modal
<img width="362" alt="Screen Shot 2023-01-31 at 22 06 33 " src="https://user-images.githubusercontent.com/11512321/215964773-02842f47-2270-4a26-a6b5-687427370546.png">

Reply:
design is working on what to do here